spec/test-data/translations/ios_generate_strings_file_from_code/sample-project/Sources/AppSampleView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct AppSampleView: View {
+    var body: some View {
+      VStack {
+        Text("app.swiftui.key1") // You should always provide a comment, not providing one like here is bad practice
```

Comment:
Nitpick.

I got confused by a comment say that the code is not providing a comment 🤔 Then I realized, looking at the line below, it refers to the `comment` argument.

How about this?

```suggestion
        // You should always provide a `comment` argument to `Text`. Not providing one like here is bad practice, we 
        // are doing it only to verify the localization implementation.
        Text("app.swiftui.key1")
```

Reply:
Addressed in 18d380e9b60ff9c4c963098633500783c93e7801 (with even slightly more descriptive wording)